src/window/window.js

```diff
@@ -93,7 +93,7 @@ export default class Window extends Component
                 <div className="content">
                 {
                     React.cloneElement(this.props.source,
-                    {id:this.state.id})
+                    {id:this.state.id, value:this.props.value})
```

Comment:
did you set this value for code in editor.js?

Reply:
yes I did. So that when we dont have any contents , we will get the `//type the code here` message.

Reply:
```javascript
<Editor
      value = {value===undefined?code : value}
/>
```

Reply:
ah thats pretty cool man Kudos for the wonderful work..... Added a lot of points (Psst already u blew everyone on top...lol).